### PR TITLE
Add Pino logging to all workers

### PR DESCRIPTION
**Pull request by JakobMiksch**
- adds pino logging to all workers
- The logs are stored in a file in a JSON structure
- Logs are also send to standard out in an colorized and prettyfied way
- The logs can be ready with CLI tools like pino-pretty as described in the README
- later they could be visualized with monitoring tools like Grafana, checkmk oder Kibana.
- connected: https://github.com/klips-project/rabbitmq-worker/pull/140

### CODE REVIEW

File: src/dispatcher/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Dispatcher Worker
+
+## Logging
+
+The dispatcher writes JSON-structured logs using the the Pino library to the path defined with the environment variable `LOG_FILE_PATH`.
+
+The logs can be viewed with a texteditor, but it is easier to read the with the tool `pino-pretty`(installation with `npm i -g pino-pretty`). Examples:
```
Comment (hblitza) [inline src/dispatcher/README.md:7]:
```suggestion
The logs can be viewed using a texteditor, but it is easier to read the with the tool `pino-pretty`(installation with `npm i -g pino-pretty`). Examples:
```

File: src/send-mattermost-message/Dockerfile [hunk 2]
```diff
@@ -9,4 +9,5 @@ WORKDIR /home/
 RUN npm install
 COPY src/send-mattermost-message/index.js worker/
 COPY src/workerTemplate.js .
+COPY src/logger.js .
 CMD ["./start-worker.sh"]
```
Comment (hblitza) [inline src/send-mattermost-message/Dockerfile:13]: missing new line

**Review by hblitza — APPROVED** [review]

**Review by weskamm — APPROVED** [review]

File: src/send-mattermost-message/Dockerfile [hunk 3]
```diff
@@ -9,4 +9,5 @@ WORKDIR /home/
 RUN npm install
 COPY src/send-mattermost-message/index.js worker/
 COPY src/workerTemplate.js .
+COPY src/logger.js .
```
Comment (JakobMiksch) [inline src/send-mattermost-message/Dockerfile:12]:
```suggestion

COPY src/logger.js .
```

File: src/send-mattermost-message/Dockerfile (hunk 2 above)
Comment (JakobMiksch) [inline src/send-mattermost-message/Dockerfile:13]:
```suggestion
CMD ["./start-worker.sh"]

```

**Pull request merged**